docs/ref/models/options.txt

```diff
@@ -326,6 +326,11 @@ The name of the manager to use for the model's
     This is a list or tuple of 2-tuples in the format ``(permission_code,
     human_readable_permission_name)``.
 
+    .. versionadded:: 1.11
```

Comment:
These annotations should go below the new text, similar to c1b6f554e405fe733e8d80f7e3d77c277810e707.

Also, please wrap doc lines at 79 characters.